### DISCUSSION TITLE
Day PV1 energy and day pv2 energy, Total PV1 Energy and Total PV2 Energy

### DISCUSSION THREAD

**Discussion by ap20132022** [kellerza/sunsynk#307] [Q&A]
Does anyone know whether it's possible to get this information?
thank you for your answer.

**Comment by kellerza** [comment]
The combined value is available in the definition file (day_pv_energy, total_pv_energy)

For individual ones you will have to check the modbus register docs for your inverter & could potentially add a custom sensor/create a PR on the defs

Since you have pv1_power and pv2_power you could also do this in HA using a Riemann sum integral sensor (integration sensor)

**Comment by ap20132022** [comment]
@kellerza : thank you very much for your answer.
I use day_pv_energy and total_pv_energy and the Riemann sum integral sensor already. The communication with inverter was made over Wireless Dongle. That's why it's not so fast. That makes the energy calculation over Riemann not really accurate.
I hope that intern the inverter it has already calculated the PV1 and PV2 to get the total energy. 

I've studied the modbus register docs but couldn't find the register for PV1 and PV2 energy.
